Add test more behave tests for livepatch

lucasmoura opened the pull request:
As stated by #1118 and #1119 , we need to create more test for livepatc/fips for both Bionic and Xenial. This PR is an initial effort towards that direction

blackboxsw commented on features/attached_enable.feature:
```diff
@@ -266,3 +266,30 @@ Feature: Enable command behaviour when attached to an UA subscription
+    Scenario Outline: Attached enable livepatch on a machine with fips active
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I run `canonical-livepatch status` with sudo
+        Then I will see the following on stderr:
```
Let's add a When/Then for running ua status and validating that stdout regexp matches fips +enabled too

lucasmoura commented:
@blackboxsw I have updated both tests and added the fips kernel test for bionic. After we solve the priority issue in Xenial, I can add it back there too.
